sdk/elasticsans/arm-elasticsan/README.md

```diff
@@ -6,7 +6,7 @@ This package contains an isomorphic SDK (runs both in Node.js and in browsers) f
 
 [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/elasticsans/arm-elasticsan) |
 [Package (NPM)](https://www.npmjs.com/package/@azure/arm-elasticsan) |
-[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-elasticsan?view=azure-node-preview) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-elasticsan) |
```

Comment:
why is this stable version link and we are release beta one ?

Reply:
service team don't want to ga this rp now https://github.com/Azure/sdk-release-request/issues/4558#issuecomment-1752500499 , so I change the version to a beta one. 
Do I need to revert this change?